src/play-item.js

```diff
@@ -42,7 +42,7 @@ const playItem = (player, delay, item) => {
     let trackEl = player.addRemoteTextTrack({ kind: 'metadata' });
 
     item.cuePoints.forEach(cue => {
-      let vttCue = new Cue(cue.time, cue.time, cue.type);
+      let vttCue = new Cue(cue.time, cue.time + 1 / 60, cue.type);
```

Comment:
@gkatsev This PR is outdated by this one here : #37 
I'll have to open up a new PR to fix the tests as that PR used the value of 1/60 seconds to add to the endTime